src/frame_codec.c

```diff
@@ -14,132 +14,132 @@
 #include "azure_uamqp_c/amqpvalue.h"
 
 #define FRAME_HEADER_SIZE 8
-#define MAX_TYPE_SPECIFIC_SIZE	((255 * 4) - 6)
```

Comment:
Yeah, we could have a script that does this auto-magically. I would love that. But did not put much thought into that yet.